yarn-project/aztec-nr/aztec/src/note/lifecycle.nr

```diff
@@ -9,14 +9,19 @@ use crate::note::{
     note_interface::NoteInterface,
     utils::compute_inner_note_hash,
 };
-use crate::oracle::notes::{notify_created_note, notify_nullified_note};
+use crate::oracle::{
+    notes::{notify_created_note, notify_nullified_note},
+    get_public_key::get_public_key,
+};
 use crate::constants_gen::EMPTY_NULLIFIED_COMMITMENT;
+use crate::log::emit_encrypted_log;
```

Comment:
Some of these imports seem to be redundant now.

Reply:
In this case you are right. Addressed in https://github.com/AztecProtocol/aztec-packages/pull/2926/commits/c1d620f77ee157d072bd29ac41292ecf05713f8c